packages/webapp/pages/posts/[id]/index.tsx

```diff
@@ -182,7 +182,7 @@ PostPage.layoutProps = {
 export default PostPage;
 
 export async function getStaticPaths(): Promise<GetStaticPathsResult> {
-  return { paths: [], fallback: true };
+  return { paths: [], fallback: 'blocking' };
```

Comment:
This will make sure we see hydration issues in development, only caveat is that first ever request in production will now stale for the amount of time it takes to fetch data from our API, and now it would show empty screen.